docs/source/user-guide/general/workspace_settings.rst

```diff
@@ -0,0 +1,39 @@
+=======================
+Workspace Settings & Users
+=======================
+
+The Workspace Settings is here to help the adimn control the users, models and subscription plan.
+This is also an easy way of separating the monitoring data in cases where you have completely independent teams, 
+with a significant number of models to monitor.
+
+Invite a user
+--------------
+ You can invite new users via the "Invite Users" button:
+
+.. image:: /_static/images/user-guide/general/invite_users.png
+    :width: 600
+
+Workspace Users Tab
+-------------------
+
+Member roles assignment
+~~~~~~~~~~~~~~~~~~~~~~~
+ Here you can edit users role to grant users various permissions.
+
+.. image:: /_static/images/user-guide/general/edit_user.png
+    :width: 600
+
+Model access setting
+~~~~~~~~~~~~~~~~~~~~
+ Here you can edit users the models each users has access to and what models the users will view.
```

Comment:
I would write instead: "Here you can assign model access permissions to users. When you need to add/remove access of multiple models to a single user, this is the best place to perform the operations."